src/FindSquare/square.py

```diff
@@ -0,0 +1,23 @@
+def calculate_square(number):
+    return number ** 2

```

Comment:
Could you please provide the documentation for it.
Since we are working on an educational project it's important to provide docstrings for each part of our code.

Reply:
I've made changes to the code